### PR TITLE
IA-3869 remove storage account ref

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-3869

Tested on a new workspace

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by jordansafer — APPROVED** [review]
LGTM!

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala [hunk 1]
```diff
@@ -274,27 +274,19 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
     )
     for {
       ctx <- ev.ask[AppContext]
-      storageAccountOpt <- wsmDao.getWorkspaceStorageAccount(params.workspaceId, auth)
-      storageAccount <- F.fromOption(
-        storageAccountOpt,
-        new RuntimeException(s"${params.workspaceId} doesn't have a storage account provisioned properly")
-      )
       resp <- wsmDao.createStorageContainer(
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala:277]: Checking my understanding -- does this call fail with a permission error, now that WSM is querying the LZ? (And we have a meeting tomorrow to talk about this?)

**Review by rtitle — APPROVED** [review]
The changes look good, modulo the permission issue in the WSM call

**Comment by Qi77Qi** [comment]
@rtitle yea, things won't work until 1. permission is fixed in WSM or LZ 2. MRG tagging issue...but this change will be needed regardless

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala (hunk 1 above)
Comment (Qi77Qi) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala:277]: yep..exactly!

**Pull request merged**